packages/eslint-config/base.js

```diff
@@ -55,7 +55,12 @@ module.exports = [
         // Workaround until import supports flat config
         // https://github.com/import-js/eslint-plugin-import/issues/2556
         espree: [".js", ".cjs", ".mjs", ".jsx"],
+        "@typescript-eslint/parser": [".ts", ".tsx"],
```

Comment:
I think this might have been resolved https://github.com/import-js/eslint-plugin-import/pull/3018